packages/react-components-pixi-7/package.json

```diff
@@ -41,53 +41,49 @@
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
-    "@pixi/react-fiber": "*",
-    "@pixi/react-invariant": "*",
-    "@pixi/react-modular": "*",
-    "@pixi/math": "^7.0.0",
     "@pixi/app": "^7.0.0",
-    "@pixi/ticker": "^7.0.0",
+    "@pixi/constants": "^7.0.0",
+    "@pixi/core": "^7.0.0",
     "@pixi/display": "^7.0.0",
     "@pixi/events": "^7.1.1",
-    "@pixi/core": "^7.0.0",
-    "@pixi/sprite-animated": "^7.0.0",
-    "@pixi/text-bitmap": "^7.0.0",
     "@pixi/graphics": "^7.0.0",
+    "@pixi/math": "^7.0.0",
     "@pixi/mesh-extras": "^7.0.0",
     "@pixi/particle-container": "^7.0.0",
-    "@pixi/constants": "^7.0.0",
+    "@pixi/react-fiber": "*",
+    "@pixi/react-invariant": "*",
+    "@pixi/react-modular": "*",
     "@pixi/sprite": "^7.0.0",
-    "@pixi/text": "^7.0.0",
+    "@pixi/sprite-animated": "^7.0.0",
     "@pixi/sprite-tiling": "^7.0.0",
+    "@pixi/text": "^7.0.0",
+    "@pixi/text-bitmap": "^7.0.0",
+    "@pixi/ticker": "^7.0.0",
     "@testing-library/react": "^13.4.0",
-    "canvas": "^2.11.0",
```

Comment:
Isn't this needed for tests?

Reply:
I dont see it being used anywhere?
I just see `jest-webgl-canvas-mock` being used

Reply:
I'll take a look, it was already in the original codebase

Reply:
It's actually a peer dependency of `js-dom`: https://github.com/jsdom/jsdom/blob/master/package.json#L51